test_pytest_pylint.py

```diff
@@ -65,6 +66,19 @@ def test_pylintrc_file(testdir):
     assert 'Line too long (10/3)' in result.stdout.str()
 
 
+def test_pylintrc_file_toml(testdir):
+    """Verify that pyproject.toml can be used as a pylint rc file."""
+    rcfile = testdir.makefile('.toml', """
+[tool.pylint.FORMAT]
+max-line-length = "3"
```

Comment:
With `3` instead of `"3"`, toml loads it as an integer and not a string and breaks pylint.  This might change in the future.